ROADMAP.md

```diff
@@ -32,27 +32,23 @@ Starting from scratch, create standard configuration that can be used with almos
 * Environment configuration
   - `.editorconfig`
   - `.gitignore`
-  - **Status**: Complete
 * Javascript configuration
   - `.eslintrc.js`
   - `.eslintignore`
-  - **Status**: Complete
 * Stylesheet configuration
   - Sass architecture specifications (e.g. 7+1, @use vs. @import)
   - `.stylelintrc.js`
-  - **Status**: Complete
 * Lint scripts
   - `eslint`
   - `stylelint`
-  - **Status**: Complete
 * Documentation
   - `README.md`
     - Standard data formats
   - `CHANGELOG.md`
   - Versioning strategy
   - Contribution instructions
-  - **Status**: Complete
 
+These changes were implemented in https://github.com/quicksolutions/starter-kit/pull/1.
```

Comment:
I think we should keep all of these Status lines, and add a status label to the phase headline. Something like either of the following. This link could move up to just below the headline or the "Complete" word could be a link to the PR.
`## [Complete] Phase 0: Universal Configuration`
`## Phase 0: Universal Configuration - COMPLETE`

Reply:
I wouldn't want to change headlines because they can be landing anchors for links, however I will move the line indicating completion in the PR #23